cli/inspect/inspector_test.go

```diff
@@ -175,7 +344,23 @@ func TestTemplateInspector_Flush(t *testing.T) {
 		fields  fields
 		wantErr bool
 	}{
-		// TODO: Add test cases.
+		{
+			name: "testTemplateInspectorFlushBufferLen0",
+			fields: fields{
+				outputStream: &bytes.Buffer{},
+				buffer:       new(bytes.Buffer),
+				tmpl:         tmpl1,
+			},
+			wantErr: false,
+		}, {
+			name: "testTemplateInspectorFlushBufferLenNot0",
```

Comment:
s/0/zero/

Reply:
I'm sorry for this mistake. I will change it immediately.